### ISSUE TITLE
train from scratch

### ISSUE THREAD

**Issue by mmitchef** [gordicaleksa/pytorch-deepdream#12]
Thanks for sharing your code. My goal is to use GoogleNet on face data. You mentioned that the result of the model depends on the dataset. Does this mean if I want to see components of human face  in the deep dream images I need to train the model from scratch on human face. I tried to fine-tune the model although it gave me good accuracy but it didn't give me dream-like images and if it gave me any dream like image it contained human face components. Do you have any idea or suggestion how to approach this?

**Comment by mmitchef** [comment]
I could figure out the problem.
